bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```

Comment:
remove unused import

Reply:
Dear @perrozzi I have removed unused import line. 
I don't know about the conflict under.